### PR TITLE
Post on domain adaptation

### PR DESCRIPTION
**Pull request by ghost**
Using a synthetic annotated dataset to learn segmentation on another domain of real images

### CODE REVIEW

File: deep-learning/_posts/2018-07-06-domain-shift.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+author: Sarah Leclerc
+pdf:   https://arxiv.org/abs/1711.06969
+cite:
+  authors: "Swami Sankaranarayanan, Yogesh Balaji, Arpit Jain, Ser Nam Lim, Rama Chellapa"
+  title:   "Learning from Synthetic Data : Addressing Domain Shift for Semantic Segmentation"
+  venue:   "CVPR 2018"
+---
+
+
+The authors propose an approach based on GANs to close the performance gap due to domain shift when applying a network learnt on synthetic data to real images.
```
Comment (CharlesAuthier) [inline deep-learning/_posts/2018-07-06-domain-shift.md]: 'learnt' -> 'learned' maybe?

File: deep-learning/_posts/2018-07-06-domain-shift.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Training scheme
+
+More specifically, they adress Unsupervised Domain Adaptation, when the data is composed of two sets : the source domain - for which we possess segmentation maps - and the target domain - that we would like to segment. This could at end be used to create ground truth for unlabeled datasets.
```
Comment (CharlesAuthier) [inline deep-learning/_posts/2018-07-06-domain-shift.md]: at end -> at the end ?

Comment (CharlesAuthier) [inline deep-learning/_posts/2018-07-06-domain-shift.md]: adress with one "d" is in french in english is two.

File: deep-learning/_posts/2018-07-06-domain-shift.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+They apply this  on :
+annotated synthetic data : SYNTHIA (9400 images of a virtual city) , GTA5 (24966 images from the game)
+real data : CITYSCAPES (5000 urban street images from a moving vehicle - 2975 for training and 1595 for testing)
+
+Here are the results on the 16 / 19 shared classes :
+
+![](/deep-learning/images/domain-shift/res.png)
+
+On each experiment, the supervision by a generative network improved results. They also show that each component of their final loss is beneficial and that the improvement is consistent with image size.
```
Comment (jodoin) [inline deep-learning/_posts/2018-07-06-domain-shift.md]: For each

**Pull request merged**